Gemfile

```diff
@@ -1,3 +1,3 @@
 source "https://rubygems.org"
 
-gem "cardboard", "~> 2.1"
+gem "cardboard23", "~> 2.1"
```

Comment:
I'm not sure what this is? Is it debugging? Should it be committed?

Reply:
Yes, It's debugging code...